src/services/DiscordService.ts

```diff
@@ -0,0 +1,195 @@
+import profiles from 'decentraland-gatsby/dist/utils/loader/profile'
+import { ChannelType, Client, EmbedBuilder, GatewayIntentBits } from 'discord.js'
+
+import { capitalizeFirstLetter } from '../clients/utils'
+import { profileUrl } from '../entities/Profile/utils'
+import { ProposalType } from '../entities/Proposal/types'
+import { isGovernanceProcessProposal, proposalUrl } from '../entities/Proposal/utils'
+import UpdateModel from '../entities/Updates/model'
+import { UpdateAttributes } from '../entities/Updates/types'
+import { getPublicUpdates, updateUrl } from '../entities/Updates/utils'
+
+const CHANNEL_ID = process.env.DISCORD_CHANNEL_ID
+const TOKEN = process.env.DISCORD_TOKEN
+
+const client = new Client({ intents: [GatewayIntentBits.Guilds] })
+
+const DEFAULT_AVATAR = 'https://decentraland.org/images/male.png'
```

Comment:
Don't we have a "more neutral" avatar to use as a default? To avoid using male or female avatars.

Reply:
I took as reference the default avatar defined in dcl-gatsby

https://github.com/decentraland/decentraland-gatsby/blob/master/src/components/User/Avatar.tsx